staging/src/k8s.io/apiserver/pkg/storage/etcd/etcd_helper.go

```diff
@@ -380,11 +380,16 @@ func (h *etcdHelper) decodeNodeList(nodes []*etcd.Node, filter storage.FilterFun
 	}
 	for _, node := range nodes {
 		if node.Dir {
-			trace.Step("Decoding dir " + node.Key + " START")
+			// NOTE: we no longer log this as a discrete step for each directory
+			// node as it produces an immense amount of log spam for any server
+			// with a reasonable amount of content.  that spam can cause restart
```

Comment:
Can you tighten up this message to be more concrete when you fix the build problem?